### ISSUE TITLE
Error when using ReactWebChat component in a SPFX solution

### ISSUE THREAD

**Issue by suryamishra-trn** [microsoft/BotFramework-WebChat#2033]
## Version
BotFramework-Webchat version 4.4.1
NPM package

## Describe the bug
**Get an error in the SPFX web part like this:**

[SPLoaderError.loadComponentError]:
***Failed to load component "0a94872b-d988-4bcd-8ffb-0f94117e69c3" (O365HelperBotWebPart).
Original error: ***Failed to load entry point from component "0a94872b-d988-4bcd-8ffb-0f94117e69c3" (O365HelperBotWebPart).
Original error: Error loading https://component-id.invalid/0a94872b-d988-4bcd-8ffb-0f94117e69c3_0.0.1
	Cannot find module "mime/types/standard.json"

***INNERERROR:
***Failed to load entry point from component "0a94872b-d988-4bcd-8ffb-0f94117e69c3" (O365HelperBotWebPart).
Original error: Error loading https://component-id.invalid/0a94872b-d988-4bcd-8ffb-0f94117e69c3_0.0.1
	Cannot find module "mime/types/standard.json"
***CALLSTACK:
Error
    at SPLoaderError.SPError [as constructor] (https://localhost:4321/temp/workbench-packages/@microsoft_sp-loader/dist/sp-loader-assembly_default.js:8671:24)
    at new SPLoaderError (https://localhost:4321/temp/workbench-packages/@microsoft_sp-loader/dist/sp-loader-assembly_default.js:4127:28)
    at Function.ErrorBuilder.buildErrorWithVerboseLog (https://localhost:4321/temp/workbench-packages/@microsoft_sp-loader/dist/sp-loader-assembly_default.js:3786:21)
    at Function.ErrorBuilder.buildLoadComponentError (https://localhost:4321/temp/workbench-packages/@microsoft_sp-loader/dist/sp-loader-assembly_default.js:3694:21)
    at https://localhost:4321/temp/workbench-packages/@microsoft_sp-loader/dist/sp-loader-assembly_default.js:6939:47

**Part of the Error when doing Gulp Serve:**

[17:55:02] Error - [webpack] 'dist':
./node_modules/botframework-webchat-core/lib/utils/mime-wrapper.js
Module not found: Error: Can't resolve 'mime/types/other.json' in 'C:\Users\SURI\Workspace\TRN Digital\Bot Demo\O365HelperBotSPFX\node_modules\botframework-webchat-core\lib\utils'
resolve 'mime/types/other.json' in 'C:\Users\SURI\Workspace\TRN Digital\Bot Demo\O365HelperBotSPFX\node_modules\botframework-webchat-core\lib\utils'
  Parsed request is a module
  using description file: C:\Users\SURI\Workspace\TRN Digital\Bot Demo\O365HelperBotSPFX\node_modules\botframework-webchat-core\package.json (relative path: ./lib/utils)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: C:\Users\SURI\Workspace\TRN Digital\Bot Demo\O365HelperBotSPFX\node_modules\botframework-webchat-core\package.json (relative path: ./lib/utils)
    resolve as module
      

## To Reproduce
1. Create a new SPFX web part.
2. npm install botframework-webchat
3. Use the ReactWebChat component in the spfx web part.

## Expected behavior
The steps mentioned in the github page for using react web chat should work without issues.

**Comment by corinagum** [comment]
This looks to be unrelated to SPFX and is actually a duplicate of #2019. @suryamishra-trn could you confirm?

**Comment by suryamishra-trn** [comment]
Yes I can confirm it's a duplicate of 2019 now. But on the SPFX part - there really is scarce documentation of using Bot Framework from SPFX solutions. The sample in the repo errors out during build. So the nearest example of implementation is the REACT sample from BotFramework-Webchat page. And hence I titled this as SPFX. I wouldn't be surprised if other devs come across this error while creating SPFX solutions to consume the Bot Framework.

**Comment by corinagum** [comment]
Hi @suryamishra-trn, anyone trying to use Web Chat within the past two weeks or so should have come across this bug - it is unrelated to whether or not the user was trying to use SPFX as well. that bug has been resolved and a bump to 4.4.2 will allow Web Chat to work as expected. 

Regarding SPFX support, we currently have no documentation verifying whether or not Web Chat and SPFX are compatible. I just started discussions with the team about verifying SPFX compatibility with Web Chat and hopefully creating a blog post of information. @v-kydela is the man to track.  

Closing this issue since #2019 has been resolved.
